shell/common/benchmark_BigBenchSchedule.sh

```diff
@@ -24,7 +24,7 @@ BENCH_EXTRA="throughput schedule"
 # Check supplied benchmarks
 check_bench_list
 
-if [ ! $user_suplied_bench_list ]; then
+if [ ! $user_supplied_bench_list ]; then
```

Comment:
Good to fix the typo, still needs quotes around the variable to avoid a syntax error if `$user_supplied_bench_list` is empty. Or use `[[`, if you prefer.